server/src/Controllers/ProjectsController.cs

```diff
@@ -25,9 +26,17 @@ public class ProjectsController(ProjectsService projects) : ControllerBase
         private readonly ProjectsService projects = projects;
 
         [HttpGet]
-        public IActionResult Fetch()
-        {
-            return Ok(projects);
+        public async Task<IActionResult> Fetch(string? search, string? orderBy, string? sort, int? size, int? page) {
```

Comment:
Can you update this to only fetch projects for the current user. Not all projects. Just current user projects.

Reply:
I'll work on that next.

Reply:
Alright. Pls make it part of this pull request.